packages/progress-bar/src/Kaizen/ProgressBar.elm

```diff
@@ -94,7 +94,7 @@ view (Config config) =
             [ classList [ ( .progressBackground, True ) ] ]
             [ div
                 [ class (moodToClass config.mood)
-                , classList [ ( .animating, isAnimating |> Debug.log "isAnim" ) ]
+                , classList [ ( .animating, isAnimating ) ]
```

Comment:
I'm pretty sure Debug.log isn't required here, but I wonder if the value is supposed to be "inAnim" (a string) rather than "isAnimating" (a boolean).

@mbylstra feel like answering an elm question?

Reply:
Oops, this looks like it was accidentally left in and can definitely be removed! 

(FWIW, in `|> Debug.log "isAnim"`, the pipeline operator puts the value on the left as the second argument to the Debug.log function, so it'd be logged as `isAnim True` or `isAnim False`. It's pretty handy for quickly putting a log statement in around an expression - something I miss from Elm. I often make a function similar to this in JS to half replicate it... eg "`function logExpression(exp) { console.log exp; return exp}`". We didn't have the no logging to console linter warning in Elm though 😅 )